common/src/main/resources/schema/development/schema.json

```diff
@@ -167,17 +170,123 @@
         }
       }
     },
-    "storageObject": {
+    "azureStorageObject": {
       "type": "object",
       "properties": {
         "region": {
-          "enum": ["southamerica-west1", "us-central1", "us-east1", "us-east4", "us-west1", "us-west4", "europe-north1", "europe-west1", "europe-west4", "asia-east1", "asia-southeast1"]
+          "enum": [
+            "eastus",
```

Comment:
Kind of a bummer that these all need to be manually specified. I wonder how often these change?

Reply:
My guess is that they change pretty rarely, and are likely added to rather than removed from if they are. (Just because I think it is unlikely they will take down servers with people's data on them)

I agree that it is a bummer though. It is possible we could template it out and generate the json-schema at build time, but that would be a bigger change I think. (I also don't know if there are apis to get these lists automatically or not)

Reply:
I was wondering if there was an API as well.

Reply:
Looks like there might be? https://stackoverflow.com/questions/44143981/is-there-an-api-to-list-all-azure-regions

Reply:
Neat! Why don't I add a ticket to look into schema templating to our backlog

Reply:
Added: https://broadworkbench.atlassian.net/browse/DC-624